### PR TITLE
[WIP] basic tutorial

### PR DESCRIPTION
**Pull request by Hearen**
#8 will be used

### CODE REVIEW

**Comment by Hearen** [comment]
@ralf0131 The basic framework will be like this, what do you say? 

- the demo good enough or should be further simplified?
- all commands will be covered in basic level using the demo - sounds good to you?
- a complete video will also be enclosed to represent the whole operations #8

**Comment by Hearen** [comment]
@ralf0131 Let me know if it's good to you then I can start to get it done. 

Your reply will be appreciated :)

File: site/src/site/sphinx/en/tutorial-for-beginners.md [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+package demo;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+public class Demo {
+    static class Counter {
+        private static AtomicInteger count = new AtomicInteger(0);
+        public synchronized static void increment() {
```
Comment (jamesxuhaozhe) [inline site/src/site/sphinx/en/tutorial-for-beginners.md]: count is already atomic and thread safe，not sure why we still need synchronized keyword?

File: site/src/site/sphinx/en/tutorial-for-beginners.md [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+        list2.add("c");
+        list2.add("d");
+
+        int len = addTwoLists(list, list2);
+    
+    }
+
+    private synchronized static <T> int addTwoLists(List<T> list1, List<T> list2) {
+        list1.addAll(list2);
+        return list1.size();
```
Comment (jamesxuhaozhe) [inline site/src/site/sphinx/en/tutorial-for-beginners.md]: doesn't seem like these two methods need synchronized keyword, list1 and list2 both are local variable and don't need to be guarded

File: site/src/site/sphinx/en/tutorial-for-beginners.md [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+        public synchronized static void increment() {
+            count.incrementAndGet();
+        
+        }
+        public synchronized static int value() {
+            System.out.println("Processing...");
+            return count.get();
+        
+        }
+
```
Comment (jamesxuhaozhe) [inline site/src/site/sphinx/en/tutorial-for-beginners.md]: redundant line

File: site/src/site/sphinx/en/tutorial-for-beginners.md [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+public class Demo {
+    static class Counter {
+        private static AtomicInteger count = new AtomicInteger(0);
+        public synchronized static void increment() {
+            count.incrementAndGet();
+        
+        }
+        public synchronized static int value() {
```
Comment (jamesxuhaozhe) [inline site/src/site/sphinx/en/tutorial-for-beginners.md]: same question as above, synchronized maybe removed

File: site/src/site/sphinx/en/tutorial-for-beginners.md [hunk 5]
```diff
@@ -0,0 +1,282 @@
...
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+public class Demo {
+    static class Counter {
+        private static AtomicInteger count = new AtomicInteger(0);
+        public synchronized static void increment() {
+            count.incrementAndGet();
+
```
Comment (jamesxuhaozhe) [inline site/src/site/sphinx/en/tutorial-for-beginners.md]: empty line

**Comment by hengyunabc** [comment]
@jamesxuhaozhe 现在已经有在线教程了，所以这个PR关掉。
https://alibaba.github.io/arthas/arthas-tutorials

**Pull request closed without merging**